pkg/actions/lua/storage/aws/s3.go

```diff
@@ -121,7 +121,7 @@ func deleteRecursive(c *S3Client) lua.Function {
 				errs = errors.Join(errs, errDel)
 			}
 
-			if !listObjects.IsTruncated {
+			if !*listObjects.IsTruncated {
```

Comment:
Can you use the aws.* helper function to extract the value as it will check for nil

Reply:
yes